### ISSUE TITLE
[TabLayout] ripple shape is different from spec

### ISSUE THREAD

**Issue by manabu-nakamura** [material-components/material-components-android#4403]
**Description:** The ripple shape of TabLayout is different from the spec.
Catalog app:
- circle with top and bottom cut off?
![Screenshot_20241124_233441](https://github.com/user-attachments/assets/7e58bbf4-2be4-48fd-8c4f-4c92600c10b2)
https://m3.material.io/components/tabs/guidelines:
- rectangle
![m2k2312r-18](https://github.com/user-attachments/assets/7aaa7e4f-c87a-4157-b522-e04497da9312)

**Expected behavior:**
- rectangle
![Screenshot_20241124_233630](https://github.com/user-attachments/assets/92655f4e-20c6-4b01-94fc-458d5a1a36f3)

**Source code:** https://github.com/material-components/material-components-android/tree/master/catalog/java/io/material/catalog/tabs

**Android API version:** 35

**Material Library version:** 1.13.0-alpha08

**Device:** Emulator 35.2.10

**See also:** #3157

**Comment by manabu-nakamura** [comment]
The ripple shape of BottomNavigationView (1.6.1 and below) was similar to the ripple shape of TabLayout (https://github.com/material-components/material-components-android/commit/11578b34f3cd37ac57c4c8856850f385bb338d2e).
![Screenshot_20241125_183501](https://github.com/user-attachments/assets/0420cfcf-b24f-48ae-b1e9-2f9be86ba013)

**Comment by manabu-nakamura** [comment]
Jetpack Compose:
![Screenshot_20241126_141058](https://github.com/user-attachments/assets/fc985978-8d23-4885-9215-89bfe74ee575)

**Comment by imhappi** [comment]
Closed in d68ebbebdf08f7cf320dee7b9dd36398516935db

**Comment by manabu-nakamura** [comment]
Workaround: put into your app's res/values/styles.xml:
```xml
<resources>
    <style name="Theme.App" parent="Theme.Material3.*">
        <item name="tabStyle">@style/Widget.App.TabLayout</item>
    </style>

    <style name="Widget.App.TabLayout" parent="Widget.Material3.TabLayout">
        <item name="tabUnboundedRipple">false</item>
    </style>
</resources>
```
https://github.com/material-components/material-components-android/blob/master/docs/components/Tabs.md
